.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  location: ''
+  subscription: ''
+  azureDevOpsJSONSPN: ''
```

Comment:
just wonder if this would be better being called azureDevOpsE2EJSONSPN?

Reply:
Makes sense. I made the change